src/System.ComponentModel.Primitives/src/System.ComponentModel.Primitives.csproj

```diff
@@ -9,19 +9,23 @@
     <RootNamespace>System.ComponentModel.Primitives</RootNamespace>
     <AssemblyName>System.ComponentModel.Primitives</AssemblyName>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
-    <PackageTargetFramework>dotnet5.1</PackageTargetFramework>
```

Comment:
As with the others this should be conditioned instead of deleted, and net46 should have IsPartialFacadeAssembly=true set.